operations/availabilityblocks.md

```diff
@@ -66,231 +62,222 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
-| `CompanyId` | string | optional | Unique identifier of the [Company](companies.md#company) linked to the block. |
-| `Budget` | [Currency value](accountingitems.md#currency-value) | optional | The tentative budget for the total price of reservations in the block. |
-| `State` | string [Availability block state](#availability-block-state) | required | State of the availability block. |
-| `ReservationPurpose` | string [Reservation purpose](reservations.md#reservation-purpose) | optional | The purpose of the block. |
+| `EnterpriseId` | string | required | Unique identifier of the [enterprise](enterprises.md#enterprise). |
+| `ServiceId` | string | required | Unique identifier of the `Service` the block is assigned to. |
+| `RateId` | string | required | Unique identifier of the `Rate` the block is assigned to. |
+| `VoucherId` | string | optional | Unique identifier of the `Voucher` used to access specified private `Rate`. |
+| `BookerId` | string | optional | Unique identifier of the `Customer` on whose behalf the block was made. |
+| `CompanyId` | string | optional | Unique identifier of the `Company` linked to the block. |
+| `TravelAgencyId` | string | optional | Unique identifier of `Company`with `Travel agency contract` the Availability Block is related to. |
+| `Budget` | [Currency value (ver 2018-06-07)](_objects.md#currency-value-ver-2018-06-07) | optional | The tentative budget for the total price of reservations in the block. |
+| `State` | [Availability block state](availabilityblocks.md#availability-block-state) | required | State of the availability block. |
+| `ReservationPurpose` | [Reservation purpose](reservations.md#reservation-purpose) | optional | The purpose of the block. |
 | `CreatedUtc` | string | required | Creation date and time of the block in UTC timezone in ISO 8601 format. |
 | `UpdatedUtc` | string | required | Last update date and time of the block in UTC timezone in ISO 8601 format. |
-| `FirstTimeUnitStartUtc` | string | required | Start of the time interval, expressed as the timestamp for the start of the first time unit, in UTC timezone ISO 8601 format. See [Time units](../concepts/time-units.md). |
-| `LastTimeUnitStartUtc` | string | required | End of the time interval, expressed as the timestamp for the start of the last time unit, in UTC timezone ISO 8601 format. See [Time units](../concepts/time-units.md). |
+| `FirstTimeUnitStartUtc` | string | required | Start of the time interval, expressed as the timestamp for the start of the first [time unit](../concepts/time-units.md), in UTC timezone ISO 8601 format. |
+| `LastTimeUnitStartUtc` | string | required | End of the time interval, expressed as the timestamp for the start of the last [time unit](../concepts/time-units.md), in UTC timezone ISO 8601 format. The maximum size of time interval depends on the service's [time unit](../concepts/time-units.md): 367 hours if hours, 367 days if days, or 60 months if months. |
```

Comment:
Care is needed here, I think the original/manual version of the docs is correct.